lib/challenge.js

```diff
@@ -19,7 +19,31 @@
   // - park
 
 // Answer Starts Here
-
+//'Spending Budget'
+// has:
+// - user info
+// - total spent amount
+// - categories of things spent on
+// can:
+// - keep track of spending on a daily basis.
+// - keep track of how much money spent on each category.
+
+//'Calories'
+// has:
+// - user info
+// - meals
+// can:
+// - keep track of calories consumed each meal.
+// - keep track of the total calories consumed in a day.
+
+//'Trips'
+// has:
+// - user info
+// - cost per trip
+// - distance per trip
+// can:
+// - keep track of trips between 2 destinations.
+// - keep track of total cost.
```

Comment:
These are three very nice-looking abstractions. But how do they relate to a Calendar application?